template/{% if docs_type=="sphinx" %}docs{% endif %}/tutorials.md

```diff
@@ -0,0 +1,11 @@
+# Tutorials
+
+Tutorials for installation and typical usage. New users start here.
+
+```{toctree}
+:maxdepth: 1
+:glob:
+
+tutorials/installation
+tutorials/*
```

Comment:
@coretl this no longer appears to work, maybe due to a sphinx update?

Now it generates a warning because installation is the only document in `tutorials` so `tutorials/*` does not refer to any unclaimed documents.

What do you think is the best thing to do? I'm quite keen on leaving the `tutorials/*` pattern in the template.
The problem would nicely go away if we could think of another default tutorial.

Reply:
Unfortunately I don't think we have a second tutorial, maybe remove `installation` and leave the `*`?